assets/src/realtime.d.ts

```diff
@@ -49,6 +49,16 @@ export type RouteStatus = "on_route" | "laying_over" | "pulling_out"
 
 export type RunId = string
 
+export type TrainVehicleId = string
+
+export interface TrainVehicle {
```

Comment:
"TrainVehicle" sounds verbose. What do you think about a global find/replace "TrainVehicle" to just "Train"?

Reply:
I went back and forth on this. The big sticking point for me was that everywhere else in the world these are simply vehicles. Since we default to vehicle meaning specifically a bus I wanted to differentiate them, but I was afraid of adding confusion in the long run by divorcing them completely from the vehicle terminology.